applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -133,9 +133,9 @@ static void unicast_client_location_cb(struct bt_conn *conn, enum bt_audio_dir d
 {
 	int ret;
 
-	if (loc == BT_AUDIO_LOCATION_SIDE_LEFT) {
+	if (loc == BT_AUDIO_LOCATION_FRONT_LEFT) {
```

Comment:
Rebase

Reply:
Done the rebase, and also removed the comment above.